src/utils/key.rs

```diff
@@ -223,6 +239,29 @@ impl KeyBundle {
 
                 Ok(decrypt(cipher, &self.key, None, data)?)
             }
+            "rsa" => {
+                let rsa = Rsa::private_key_from_pem(&self.key)?;
+                if data.len() > rsa.size() as usize {
+                    return Err(RvError::ErrPkiInternal);
```

Comment:
This doesn't look like an internal error. It should be invalid input parameter or similar.

Reply:
> This doesn't look like an internal error. It should be invalid input parameter or similar.

fixed